assignment7.txt

```diff
@@ -0,0 +1,16 @@
+1)
+    ++*p increments the value that p is pointing to
+    *p++ gets the value that p is pointing to, then increments p
+    *++p increments p, then returns the new value that p is pointing at
+2) yes?
+3) You can pass by reference
```

Comment:
Pointers are more efficient in handling arrays and data tables 
They can be used to return multiple values from a function via function arguments 
Pointers permit references to functions 
Pointers allow C to support dynamic memory management 
Pointers reduce length and complexity of programs